src/librustc_target/spec/mipsel_sony_psp_linker_script.ld

```diff
@@ -0,0 +1,289 @@
+/*
```

Comment:
Re license? Does this compatible with Rustc license? Or we add this to exceptional license list?

Reply:
We do have some Fuchsia-specific dependencies under 3-BSD, but yeah, according to the terms we'd have to reproduce parts of this file in the shipped binaries. Unsure about this, so I'll hand this off to someone else for the final review.

Reply:
https://github.com/rust-lang/rust/blob/9912925c254589f58338cb2993163e618475ff75/src/tools/tidy/src/deps.rs#L33-L35

Reply:
Is there anything I can do to help move this along?

Reply:
I've [asked](https://rust-lang.zulipchat.com/#narrow/stream/231349-t-core.2Flicensing/topic/3-BSD.20dependencies.20in.20rustc/near/197081223) the core team for advice and am waiting on a response.